inference-engine/tests/functional/plugin/gpu/shared_tests_instances/single_layer_tests/deformable_convolution.cpp

```diff
@@ -0,0 +1,157 @@
+    ::testing::ValuesIn(with_bilinear_interpolation_pad),
+    ::testing::ValuesIn(with_modulated_scalar));
+
+INSTANTIATE_TEST_SUITE_P(
+    smoke_DeformableConvolution2D_ExplicitPadding, DeformableConvolutionLayerTest,
+    ::testing::Combine(
+        deformableConv2DParams_ExplicitPadding, ::testing::ValuesIn(netPrecisions),
+        ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),
+        ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),
+        ::testing::Values(InferenceEngine::Layout::ANY),
+        ::testing::Values(InferenceEngine::Layout::ANY),
+        ::testing::Values(std::vector<size_t>({1, 2, 3, 3})),
+        ::testing::Values(CommonTestUtils::DEVICE_GPU)),
+    DeformableConvolutionLayerTest::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(
+    smoke_DeformableConvolution2D_AutoPadValid, DeformableConvolutionLayerTest,
+    ::testing::Combine(
+        deformableConv2DParams_AutoPadValid, ::testing::ValuesIn(netPrecisions),
+        ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),
+        ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),
+        ::testing::Values(InferenceEngine::Layout::ANY),
+        ::testing::Values(InferenceEngine::Layout::ANY),
+        ::testing::Values(std::vector<size_t>({1, 2, 3, 3})),
+        ::testing::Values(CommonTestUtils::DEVICE_GPU)),
+    DeformableConvolutionLayerTest::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(
+    smoke_DeformableConvolution2D_DeformableGroups_ExplicitPadding, DeformableConvolutionLayerTest,
+    ::testing::Combine(
+        deformableConv2DParams_DeformableGroups_AutoPadExplicit,
+        ::testing::ValuesIn(netPrecisions),
+        ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),
+        ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),
+        ::testing::Values(InferenceEngine::Layout::ANY),
+        ::testing::Values(InferenceEngine::Layout::ANY),
+        ::testing::Values(std::vector<size_t>({1, 16, 66, 66})),
+        ::testing::Values(CommonTestUtils::DEVICE_GPU)),
+    DeformableConvolutionLayerTest::getTestCaseName);
+
+///* ============= Single Test Case ============= */
+const std::vector<std::vector<size_t>> single_deform_vals = {{1, 54, 28, 28}};
+const std::vector<std::vector<size_t>> single_kernel = {{1, 3, 3, 3}};
+const std::vector<size_t> single_deform_groups = {3};
+
+const auto deformableConv2DParams_SingleTestCase = ::testing::Combine(
+    ::testing::ValuesIn(single_deform_vals),
+    ::testing::ValuesIn(single_kernel),
+    ::testing::ValuesIn(strides),
+    ::testing::ValuesIn(padBegins),
+    ::testing::ValuesIn(padEnds),
+    ::testing::ValuesIn(dilations),
+    ::testing::ValuesIn(groups),
+    ::testing::ValuesIn(single_deform_groups),
+    ::testing::ValuesIn(numOutChannels),
+    ::testing::Values(ngraph::op::PadType::EXPLICIT),
+    ::testing::ValuesIn(with_bilinear_interpolation_pad),
+    ::testing::ValuesIn(with_modulated_scalar)
+);
+//
```

Comment:
Please add comment or remove this unnecessary code.

Reply:
done